### PR TITLE
Adding holiday message when no service

### LINKED ISSUE #47: Show when buses aren't running because of holidays
Use the calendar file to see when holidays when buses don't run and display an appropriate message.

### PR DESCRIPTION
**Pull request by tescher**
Resolves #47

NOTE: need to install moment-timezone support:

`npm install moment-timezone`

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ app.use(function(req, res, next) {
 // will print stacktrace
 if (app.get('env') === 'development') {
     app.use(function(err, req, res, next) {
+        console.log(err.message);
```
Comment (NigelKibodeaux) [inline app.js]: looks like leftover debugging code that would throw an exception if err is null

Comment (NigelKibodeaux) [inline app.js]: never mind! I just got more context.

**Comment by NigelKibodeaux** [comment]
looks good with the exception of needing that dependency added to package.json

**Comment by tescher** [comment]
I added the npm install note to the other issue. I can fix it but not tonight. I will need to set up another VM with a clean install in order to test it.

**Comment by brendanbabb** [comment]
I deployed to AWS and installed moment-timezone

**Pull request merged**